src/stylesheets/app.scss

```diff
@@ -90,14 +90,61 @@ button {
 	margin-top: 40px;
 }
 
+.confetti {
+	position: absolute;
+	z-index: 2;
+	width: 300px;
+	height: 300px;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+}
 .definition-button {
-	width: 480px;
-	height: 180px;
+	width: 300px;
+	height: 100px;
 	background-color: $powder-blue;
 	border-radius: 0.5rem;
 	color: black;
 	box-shadow: #422800 4px 4px 0 0;
-	margin: 10px;
+	margin: 15px 0;
```

Comment:
Not commenting on anythign specific here, but if there are ever values (e.g., for box-shadows or border-radiuses) that you find yourself repeating a lot, you can set them to constants like you do for colors (e.g. `$power-blue).